contracts/erc7683/ERC7683.sol

```diff
@@ -6,10 +6,10 @@ pragma solidity ^0.8.0;
 struct GaslessCrossChainOrder {
     /// @dev The contract address that the order is meant to be settled by.
     /// Fillers send this order to this contract address on the origin chain
-    address originSettler;
+    bytes32 originSettler;
```

Comment:
Does this require any changes in ERC proposal?

Reply:
yes. the ERC itself. is being updated by Matt and others.